charts/dbildungs-iam-keycloak/values.yaml

```diff
@@ -57,49 +50,49 @@ containerSecurityContext:
   seccompProfile:
     type: "RuntimeDefault"
 
-volumeConfig:
-  name: realm-config
-  mountPath: "/opt/keycloak/data/import"
-  readOnly: true
-
-volumePermissions:
-  enabled: true
-  resources:
-    limits: 
-      memory: "1Gi"
-      cpu: "500m"
-    requests: 
-      memory: "1Gi"
-      cpu: "150m"
+resources:
+  limits:
+    memory: "1Gi"
+    cpu: "500m"
+  requests:
+    memory: "1Gi"
+    cpu: "150m"
 
 startupProbe:
+  enabled: true
   httpGet:
     path: "/health/started"
     port: web
   periodSeconds: 10
   failureThreshold: 30
 
 livenessProbe:
+  enabled: true
   httpGet:
     path: /health/live
     port: web
 
 readinessProbe:
+  enabled: true
   httpGet:
     path: "/health/ready"
     port: web
 
 ingress:
   ingressClassName: nginx
+  path: /
   pathType: Prefix
-  
-  httpRelativePath: "/"
-  portnumber: 80
+  annotations:
```

Comment:
these values should in a future PR set to {} and the concrete vlaues should be given to DEV via spsh-app-deploy  (ans spsh-app-release).